documentation/slate/source/includes/welcome.md

```diff
@@ -98,7 +98,9 @@ To help you in the building of your project, we have implemented tools to query
 About the data
 --------------
 
-The street network is extracted from [OpenStreetMap](http://www.openstreetmap.org). The public transport data are provided by networks that provide their timetables as open data. Some data improvements are achieved by Kisio Digital and are published back there <https://navitia.opendatasoft.com>
+The street network is extracted from [OpenStreetMap](http://www.openstreetmap.org). The public transport data are provided by networks that provide their timetables as open data. Some data improvements are achieved by Kisio Digital and are published back there <https://navitia.opendatasoft.com>.
+
+Want to know if your city is in Navitia? Know if a special contributor is used? You can either search in [datasets](#datasets) of the different [coverages](#coverage). Or use the filter provided on our data catalog <https://navitia.opendatasoft.com>.
 
```

Comment:
Not so found of it, perharps more an output example, but this is pretty self-explanatory so not in this PR :smile: